internal/datastore/store.go

```diff
@@ -1,22 +1,22 @@
 package datastore
 
-import "github.com/SimplQ/simplQ-golang/internal/models"
+import (
+	"github.com/SimplQ/simplQ-golang/internal/models"
+	"go.mongodb.org/mongo-driver/mongo"
```

Comment:
We should not import mongo here to make sure nothing is dependent on mongo.

Reply:
Done